app/commands/decidim/kids/authorize_user.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Kids
+    # A command to authorize a user with an authorization handler.
+    class AuthorizeUser < Decidim::Verifications::AuthorizeUser
+      def call
+        return broadcast(:invalid_age) unless valid_age?
+
+        super
+      end
+
+      private
+
+      def valid_age?
+        return true unless handler.valid?
+        return true if Decidim::Kids.minor_authorization_age_attributes.blank?
+
+        Decidim::Kids.minor_authorization_age_attributes.detect do |attr|
+          begin
+            age = Time.zone.today.year - Date.parse(handler.try(attr).to_s).year
```

Comment:
If the minor birthday is in December, we will get one more year than expected in the other months.

I've found [here](https://medium.com/@craigsheen/calculating-age-in-rails-9bb661f11303) an easy way to do it:

```suggestion
            age = ((Time.zone.now - Date.parse(handler.try(attr).to_s).to_time) / 1.year.seconds).floor
```

Reply:
added a test too